src/core/qgsfileutils.cpp

```diff
@@ -0,0 +1,17 @@
+#include "qgsfileutils.h"
+
+QString QgsFileUtils::humanSize( qint64 bytes )
+{
+  QStringList list;
+  list << "KB" << "MB" << "GB" << "TB";
```

Comment:
Does this possibly need to be localized? Thinking mostly about languages with non-latin character sets.